client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -114,11 +114,27 @@
               editor-id="recommendationText"
               :procedure-id="procedureId"
               @insert="text => modalProps.handleInsertText(text)" />
-            <addon-wrapper
-              ref="addonWrapper"
-              :addon-props="addonProps"
-              hook-name="recommendation.modal"
-              :ref-component="refRecModal" />
+            <dp-modal
+              ref="recommendationModal"
+              class="recommendation-modal"
+              content-classes="u-2-of-3">
+              <div class="display--inline-block">
```

Comment:
```suggestion
              <div class="inline-block">
```

Reply:
this has been adjusted here: https://github.com/demos-europe/demosplan-core/pull/1486/commits/dfd06a2e268a73699655a893b43a847de83188b1